docs/js-web-tools/bootstrap/bootstrap.rst

```diff
@@ -0,0 +1,241 @@
+Working with Bootstrap
+======================
+By `Steve Smith`_ | Originally Published: 1 June 2015 
+
+.. _`Steve Smith`: Author_
+
+Bootstrap 3 is currently the most popular web framework for developing responsive web applications. It offers a number of features and benefits that can improve your users' experience with your web site, whether you're a novice at front-end design and development or an expert. Bootstrap is deployed as a set of HTML, CSS and JavaScript files, and is designed to help your website or application scale efficiently from phones to tablets to desktops.
+
+Getting Started
+^^^^^^^^^^^^^^^
+
+There are several ways to get started with Bootstrap. If you're starting a new web application in Visual Studio, you can choose the default starter template for ASP.NET 5, in which case Bootstrap will come pre-installed:
+
+.. image:: _static/bootstrap-in-starter-template.png
+ 
+You can also install bootstrap using one of several package managers, such as bower, npm, or NuGet. In each case, the process is essentially the same:
+
+Bower
+-----
+
+.. code-block:: console
+
+	bower install bootstrap
+	
+npm
+---
+
+.. code-block:: console
+
+	npm install bootstrap
+	
+NuGet
+-----
+
+.. code-block:: console
+
+	Install-Package bootstrap
+	
+If you're referencing your own local versions of Bootstrap, you'll need to reference them in any pages that will use it. In the default ASP.NET site template, the _Layout.cshtml file does so like this:
+
+.. image:: _static/bootstrap-in-layout.png
+
+You can also simply reference Bootstrap using its CDN (Content Delivery Network) links, which is how you may wish to have your application reference bootstrap in production in any case. Simply replace the two references above with these:
```

Comment:
we can also point to ASP.NET CDN http://www.asp.net/ajax/cdn#Bootstrap_Releases_on_the_CDN_14 and mention that you can use any other CDN as well